### ISSUE TITLE
Apps that are enabled for certain groups only are shown as disabled at `php occ app:list`

### ISSUE THREAD

**Issue by pbek** [owncloud/core#17737]
### Steps to reproduce
1. Enable app for only one group, for example "admin". (In table `oc_appconfig` the configValue for `enabled` would be `["admin"]`, everything is ok if it is `yes`.)
2. Call `php occ app:list`.

### Expected behaviour
App should be shown as enabled.

### Actual behaviour
App is shown as disabled.

### Server configuration
**Operating system**:
Ubuntu Linux 15.04
**Web server:**
Apache
**Database:**
MySQL
**PHP version:**
5.6.4
**ownCloud version:** (see ownCloud admin page)
8.2 pre alpha
**Updated from an older ownCloud or fresh install:**
update
**List of activated apps:**
Enabled:
  - bookmarks: 0.4
  - calendar: 0.7.2
  - contacts: 0.4.0.0
  - files: 1.1.9
  - files_external: 0.2.3
  - files_sharing: 0.6.2
  - files_trashbin: 0.6.3
  - files_versions: 1.0.6
  - news: 5.3.8
  - tasks: 0.7.1

Disabled:
  - encryption
  - ownbackup
  - provisioning_api
  - sharelinks
  - user_ldap
  - user_webdavauth

**The content of config/config.php:**
```json
{
    "system": {
        "instanceid": "ociuf2r3opzp",
        "trusted_domains": [
            "localhost"
        ],
        "datadirectory": "\/home\/omega\/Web\/owncloud\/data",
        "overwrite.cli.url": "http:\/\/localhost:8080",
        "dbtype": "mysql",
        "version": "8.2.0.2",
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "logtimezone": "UTC",
        "installed": true,
        "theme": "",
        "maintenance": false
    }
}
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

#### Web server error log
nothing was logged

#### ownCloud log (data/owncloud.log)
nothing was logged

**Comment by LukasReschke** [comment]
@icewind1991 I'm somewhat reluctant to modify `\OC_App::isEnabled` here, is there any public API such as  `\OCP\App\IAppManager::isEnabledForUser` for all users?

**Comment by icewind1991** [comment]
`\OCP\App\IAppManager::isInstalled` is what you're looking for

**Comment by sorglos** [comment]
Is this related to this issue? https://github.com/owncloud/calendar/pull/502#issuecomment-121938499
As long as the Calendar app is restricted to certain groups and listed as disabled in ```occ```, CalDAV doesn't work.

**Comment by MorrisJobke** [comment]
Fix is in #21724
